src/Microsoft.DotNet.Build.Tasks.net45/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "Newtonsoft.Json": "7.0.1",
-    "NuGet.Versioning": "3.4.0-beta-488",
+    "NuGet.Versioning": "3.5.0-rc-1220",
```

Comment:
I've already fixed that: https://github.com/dotnet/buildtools/commit/877a0859a5a4ddc7828b52371b023ea16d73832a